app/javascript/components/admin/site_settings/SiteSettings.jsx

```diff
@@ -31,29 +27,20 @@ export default function SiteSettings() {
                   <div className="ps-4 pe-4 pt-4">
                     <h3> Customize Greenlight </h3>
                   </div>
-                  {
-                    isLoading
```

Comment:
Why is it safe to remove the isLoading now?

Reply:
I moved the whole query into `Administration` - the check for is loading is there